### DISCUSSION TITLE
Move devices between sites

### DISCUSSION THREAD

**Discussion by ederpardeiro** [amidaware/tacticalrmm#1958] [General]
Hi Guys!

Can I move objects (server or workstantions) between sites without reinstall agent?

**Comment by jmoran-shafer** [comment]
Yes, just edit the agent

**Reply by ederpardeiro** [reply]
Hi @jmoran-shafer!

Where can I edit it? Some configuration file?

**Reply by ederpardeiro** [reply]
> Yes, just edit the agent
> […](#)
> On Sat, Aug 3, 2024 at 9:15 AM Eder Pardeiro ***@***.***> wrote: Hi Guys! Can I move objects (server or workstantions) between sites without reinstall agent? — Reply to this email directly, view it on GitHub <#1958>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BB5OO2EAJTNZWYC3SZORCXDZPT62LAVCNFSM6AAAAABL6A4NICVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGAYDSMRTGA> . You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Wow.... ok... I found it. It's so simple!!!

Thank you very much!!!!

**Comment by jmoran-shafer** [comment]
Sorry, I realized it was very specific after I sent the last message.

Right click on the agent, select the edit option (I believe it’s the first
option in the right-click selection), and then change the Site. You can
also move the agent across different clients in this site list selection,
as it shows every site.
